modules/check_in/app/controllers/check_in/v1/patient_check_ins_controller.rb

```diff
@@ -4,40 +4,11 @@ module CheckIn
   module V1
     class PatientCheckInsController < CheckIn::ApplicationController
       def show
-        check_in = CheckIn::PatientCheckIn.build(uuid: params[:id])
```

Comment:
I might be a bit of a sticker here too, but wondering if you can remove v1 incrementally to avoid the dangerbot failure. I'm thinking the first iteration can be the changes to the controllers + corresponding specs, then probably the services classes (broken up into separate PRs). I think this one will be pretty straight forward to break apart